src/genericobject.cpp

```diff
@@ -182,3 +182,15 @@ std::string gob_cmd_update_nametag_attributes(video::SColor color)
 	writeARGB8(os, color);
 	return os.str();
 }
+
+std::string gob_cmd_update_infant(u16 id, u8 type, std::string client_initialization_data)
```

Comment:
@nerzhul I can't do so. Because at content_sao side its:
'os<<serializeLongString(gob_cmd_update_infant(...'
Therefore strings passed is serialized two times. One time only _init. data_, second time - _init. data_ with _id_ and _type_.